neqo-transport/src/frame.rs

```diff
@@ -426,6 +428,41 @@ impl<'a> Frame<'a> {
             d(dec.decode_varint())
         }
 
+        fn decode_ack<'a>(dec: &mut Decoder<'a>, t: u64) -> Res<Frame<'a>> {
```

Comment:
```suggestion
        fn decode_ack<'a>(dec: &mut Decoder<'a>, ecn: bool) -> Res<Frame<'a>> {
```

This might be easier for the stuff below.